eq-author/public/index.html

```diff
@@ -27,54 +62,100 @@
-          g = m[e] = function (a, b) { g.q ? g.q.push([a, b]) : g._api(a, b); }; g.q = [];
-          o = n.createElement(t); o.async = 1; o.src = 'https://' + _fs_host + '/s/fs.js';
-          y = n.getElementsByTagName(t)[0]; y.parentNode.insertBefore(o, y);
-          g.identify = function (i, v) { g(l, { uid: i }); if (v) g(l, v) }; g.setUserVars = function (v) { g(l, v) };
-          g.identifyAccount = function (i, v) { o = 'account'; v = v || {}; v.acctId = i; g(o, v) };
-          g.clearUserCookie = function (c, d, i) {
-            if (!c || document.cookie.match('fs_uid=[`;`]*`[`;`]*`[`;`]*`')) {
-              d = n.domain; while (1) {
-              n.cookie = 'fs_uid=;domain=' + d +
-                ';path=/;expires=' + new Date(0).toUTCString(); i = d.indexOf('.'); if (i < 0) break; d = d.slice(i + 1)
+        window["_fs_debug"] = false;
+        window["_fs_host"] = "fullstory.com";
+        window["_fs_org"] = window.config.REACT_APP_FULLSTORY_ORG;
+        window["_fs_namespace"] = "FS";
+        (function(m, n, e, t, l, o, g, y) {
+          if (e in m) {
+            if (m.console && m.console.log) {
+              m.console.log(
+                'FullStory namespace conflict. Please set window["_fs_namespace"].'
+              );
+            }
+            return;
+          }
+          g = m[e] = function(a, b) {
+            g.q ? g.q.push([a, b]) : g._api(a, b);
+          };
+          g.q = [];
+          o = n.createElement(t);
+          o.async = 1;
+          o.src = "https://" + _fs_host + "/s/fs.js";
+          y = n.getElementsByTagName(t)[0];
+          y.parentNode.insertBefore(o, y);
+          g.identify = function(i, v) {
+            g(l, { uid: i });
+            if (v) g(l, v);
+          };
+          g.setUserVars = function(v) {
+            g(l, v);
+          };
+          g.identifyAccount = function(i, v) {
+            o = "account";
+            v = v || {};
+            v.acctId = i;
+            g(o, v);
+          };
+          g.clearUserCookie = function(c, d, i) {
+            if (!c || document.cookie.match("fs_uid=[`;`]*`[`;`]*`[`;`]*`")) {
+              d = n.domain;
+              while (1) {
+                n.cookie =
+                  "fs_uid=;domain=" +
+                  d +
+                  ";path=/;expires=" +
+                  new Date(0).toUTCString();
+                i = d.indexOf(".");
+                if (i < 0) break;
+                d = d.slice(i + 1);
               }
             }
           };
-        })(window, document, window['_fs_namespace'], 'script', 'user');
+        })(window, document, window["_fs_namespace"], "script", "user");
       }
     </script>
   </head>
   <body>
+    <!-- Google Tag Manager (noscript) -->
+    <noscript
+      ><iframe
+        src="https://www.googletagmanager.com/ns.html?id=GTM-5STKLX7"
```

Comment:
This `id` looks specific to the app. So I wonder if it should come from an environment variable. If so it would have to follow the pattern of the other variables e.g. `REACT_APP_HOT_JAR_ID`

Reply:
It is specific, its the id of the google tag manager container

Reply:
Now that this has appeared in the source code, we should probably look to rotate it with a new Id if possible.

Reply:
Ive recreated the container now so it has a new id. Just need to set up the environment variable in concourse and then I can give it a test again